### ISSUE TITLE
Native animation (Large title) isn't scrolling properly

### ISSUE THREAD

**Issue by iDuuck** [react-navigation/navigation-ex#318]
**Current Behavior**

When scrolling through the screens in my example (see "How to reproduce"), the large title bar is bouncing. I am not sure on how to **properly** recreate the issue, because it sometimes happens with `ScrollView`, and sometimes it is happening with `FlatList`, but you can check out the example.

For me personally, the `ScrollView` on the `Home` component, isn't scrolling properly, since the title bar scrolls away faster, than the actual scroll momentum.

**Additionally:** Don't know, if I should create a new issue for this one, but I am sure, that there is a scrolling behavior, if the large title is scrolled to the half, that it bounces back to the nearest position (small title, or full large title – not cutted large title).

**Expected Behavior**

I expect it to function (scroll) properly.

**How to reproduce**

https://snack.expo.io/r1UIKryGI

**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 | iOS | 
| @react-navigation/native       | ^5.0.0-alpha.29 | 
| @react-navigation/native-stack | ^5.0.0-alpha.29 |
| react-native-screens           | 2.0.0-alpha.12 |
| react-native                   | https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz |
| expo                           | 36.0.0 |
| node                           | v12.8.0 |
| npm or yarn                    | 1.19.2 |

**Comment by evanheckert** [comment]
This seems likely related - I notice when I scroll down but release before it has completely transitioned to the small title, there's no momentum scrolling - it just stops.

**Comment by iDuuck** [comment]
@evanheckert Yeah, that's what I also observed (See my *Additional* note). Luckily  the base bug I reported here is only present on the iOS Simulator. Works like a charm on my iOS device.

**Comment by darwev0** [comment]
**I noticed the following issues:**
_Running on device: iPhone 11 Pro (iOS 13)_

- Title doesn't snap, instead it remains cut on the header (is it the expected behaviour?)
![IMG_4DB75B7FAA29-1](https://user-images.githubusercontent.com/38386337/73533612-8a1c3200-441f-11ea-8409-69b1bd4f2afb.jpeg)

**Also, when the scroll content is bigger than the screen size:**
- Title velocity when scrolling up is too fast (doesn't follow the gesture), it looks like both the Title and the ScrollView begin scrolling and their speed adds up
- When using `headerTranslucent: true` the same happens on scrolling down

Starting from your snack example i produced:
https://snack.expo.io/@darwev0/createnativestacknavigator-react-navigation-318
Home (without `headerTranslucent: true`): doesn't snap, scrolling up velocity messed up
Profile (`headerTranslucent: true`): doesn't snap, scrolling up/down velocity messed up

Could this be related to [this issue](https://stackoverflow.com/questions/47649375/ios-11-large-navigation-bar-title-unexpected-velocity)?
They're suggesting using UITableViewController instead of UIScrollView on the native side.

**Comment by iDuuck** [comment]
Exactly this!

**Comment by iDuuck** [comment]
Any update on this case? 

@satya164 @osdnk (Sorry for linking, but I suppose this is a key feature when using the native-stack).

**Comment by florand0m** [comment]
Same issue here 😞w/ the latest react-navigation v5 out!! 
Any update?

**Comment by iDuuck** [comment]
@florand0m I **suppose** this is an error with Expo only. Because Expo is limited to a specific version of `react-native-screens` and the version `react-navigation@v5` want to use, is not compatible with the latest Expo version.

**Comment by darwev0** [comment]
> @florand0m I **suppose** this is an error with Expo only. Because Expo is limited to a specific version of `react-native-screens` and the version `react-navigation@v5` want to use, is not compatible with the latest Expo version.

Sadly I've been trying with a bare project and `"react-native-screens": "^2.0.0-beta.2"` but the issue is still there.
